R/ash.R

```diff
@@ -405,10 +418,16 @@ compute_lfsr = function(NegativeProb,ZeroProb){
 #of the loglik for $\pi=(\pi_0,...,1-\pi_0,...)$ with respect to $\pi_0$ at $\pi_0=1$.
 gradient = function(matrix_lik){
   n = nrow(matrix_lik)
-  grad = n - colSums(matrix_lik/matrix_lik[,1])
+  grad = n - ColsumModified(matrix_lik)
```

Comment:
would it work to just add 1e-100 to every entry of matrix_lik?
it would be much simpler... m = matrix_lik+1e-100; grad= n- colSums(m/m[,1])

do you have a test example that illustates this problem and could be used as a unit test?